boosters/node-beginner/migrations/20200113152137-repo-association.js

```diff
@@ -0,0 +1,19 @@
+"use strict";
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.addColumn("Repos", "UserId", {
+      type: Sequelize.UUID,
```

Comment:
I think the type should be an integer since that's how the user id is generated.

Reply:
Good catch. Instead, I changed the models to use UUIDs instead. Rational is that since the ID is being returned to the users to perform further queries (against the `/users/:userid` and `/repos/:repoid` endpoints), better to use UUIDs rather than an auto incrementing integer.